ecosystem/sep-0008.md

```diff
@@ -62,106 +72,184 @@ approval_criteria="The goat approval server will ensure that transactions are co
 - If a transaction was revised, ALWAYS explain the changes that were made to a transaction through the `message` parameter.
+- Wallet should inspect revised transactions and alert the user if any of the original operations is changed.
 - Core operations shouldn't be modified as that can be confusing and misleading. For example, if the user wishes to put an offer for 1000 GOATS but due to velocity limits they can only put an offer for 500 GOATS, it is better to error with a message than to change the amount.
 - Adding an upper timebound to a transaction can help the issuer ensure that their view of the world does not get out of sync.
-- Issuers can enforce additional fees by adding additional operations. For example, any transaction involving goats, will also send 0.1 GOAT to Boris’ account.
-- Once a transaction has been signed, the issuer should mark it as pending and take it into account, as long as it hasn’t been timed out, so that they can have a accurate view of the world.
+- Issuers can enforce additional fees by adding additional operations. For example, any transaction involving GOATs, will also send 0.1 GOAT to Boris’ account.
 
 ## Account Setup
 
 Implementing Regulated Assets requires the participating accounts to be “managed” by the issuer. This is achieved by having the issuer be a co-signer on the account, such that a medium threshold operation cannot be submitted without the issuer’s approval.
 
-In the future, this requirement can be replaced by introducing protocol level [CoSigned assets](https://github.com/stellar/stellar-protocol/issues/146).
-
 ## Discussion
 
 ### Should my asset be a regulated asset?
 
 Ideally, no. Implementing Regulated Assets should only be used when absolutely necessary, such as for regulatory reasons. It comes with a substantial operational overhead, added complexity, and burden for users. Issuers should only go down this route if it is absolutely required.
-Alternatively, some other available options are to utilize [SEP006](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md) in order to perform KYC on deposit and withdraw, and/or use [AUTHORIZATION_REQUIRED](https://www.stellar.org/developers/guides/issuing-assets.html#requiring-or-revoking-authorization) which allows issuers to authorize specific accounts to hold their issued assets.
 
 ### Why doesn’t the approval service submit transactions to the network?
 
 - Separation of concerns between signing transactions and submitting them.
 - Transactions might require more signatures from further regulated asset issuers.
+- Wallets can handle the failure from transaction submission based on the type of errors and reconstruct the transaction.
+- Scaling concerns as having the approval server submit transactions would require more infrastructure.
 
+[SEP-1 stellar.toml]: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md
+[authorization flags]: #authorization-flags
+[Approval Server]: #approval-server
```

Comment:
Not a big deal if we keep it but I don't think this adds any value to the document.

Reply:
Yeah I like it because people can easily jump to the relevant section from `Overview` if they would like to dig into it.

Reply:
Oh I see what you did. Couldn't you put the links directly in the overview section? :
```
Implementing a regulated asset requires these parts:
- Issuer account with appropriate [authorization flags](#authorization-flags) set.
```

Reply:
I'll leave how you want to do this to you, LGTM

Reply:
> Couldn't you put the links directly in the overview section?

I can certainly do that. Putting it at the end of the file is useful when we have multiple references, and it makes the `raw` version easier to read, especially when the link is long. When I do it, I tend to put all links together in the bottom regardless the number of references and the length. 😃